packages/casbin/README.md

```diff
@@ -0,0 +1,143 @@
+# Casbin Middleware for Hono
+
+This is a third-party [Casbin](https://casbin.org) middleware for [Hono](https://github.com/honojs/hono).
+
+This middleware can be used to enforce authorization policies defined using Casbin in your Hono routes.
+
+## Installation
+
+```bash
+npm i hono @hono/casbin casbin
+```
+
+## Configuration
+
+Before using the middleware, you must set up your Casbin model and policy files.
+
+For details on how to write authorization policies and other information, please refer to the [Casbin documentation](https://casbin.org/).
+
+### Example model.conf
+
+```conf
+[request_definition]
+r = sub, obj, act
+
+[policy_definition]
+p = sub, obj, act
+
+[role_definition]
+g = _, _
+
+[policy_effect]
+e = some(where (p.eft == allow))
+
+[matchers]
+m = r.sub == p.sub && r.obj == p.obj && r.act == p.act
+```
+
+### Example policy.csv
+
+```csv
+p, alice, /dataset1/*, *
+p, bob, /dataset1/*, GET
+```
+
+## Usage with Basic HTTP Authentication
+
+You can perform authorization control after Basic authentication by combining it with `basicAuthorizer`.
+(The client needs to send `Authentication: Basic {Base64Encoded(username:password)}`.)
+
+Let's look at an example.
+Use the `model` and `policy` files from the [Configuration](#configuration) section.
+You can implement a scenario where `alice` and `bob` have different permissions. Alice has access to all methods on `/dataset1/test`, while Bob has access only to the `GET` method.
+
+```ts
+import { Hono } from 'hono'
+import { basicAuth } from 'hono/basic-auth'
+import { newEnforcer } from 'casbin'
+import { casbin } from '@hono/cabin'
```

Comment:
It's a typo. Should be `@hono/casbin'`.

Reply:
Fixed!
https://github.com/honojs/middleware/pull/676/commits/d0325f1b31bb097eb73ae02120dc5cdc74108bd2